_posts/2023-10-08-amazon_bedrock_part1.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: postv2
+font: serif
+title: "Amazon Bedrock Anatomy"
+description: "Part1: What's Amazon Bedrock and what it does?"
```

Comment:
Part1: What's Amazon Bedrock and what it does?
should probably be
Part 1: What is Amazon Bedrock And What Does It Do?